src/devices/BuildHat/Models/LedMode.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-namespace BuildHat.Models
+namespace Iot.Device.BuildHat.Models
```

Comment:
does this need sample update?

Reply:
At the moment, it appears to build fine.